pkg/symbol/elfutils/elfutils.go

```diff
@@ -126,32 +114,9 @@ func IsSymbolizableGoObjFile(path string) (bool, error) {
 	return false, errors.New("failed to detect .gopclntab section or section has no bits")
 }
 
-// IsGoObjFile checks whether the specified executable or library file is generated by Go toolchain.
-func IsGoObjFile(path string) (bool, error) {
```

Comment:
I assume this has not been used anywhere else, right?

Reply:
Yeah, I couldn't find any references in parca and parca-agent repositories.